svir/test/integration/test_drive_oq_engine.py

```diff
@@ -171,12 +174,90 @@ def load_output(self, calc, output):
             dlg = OUTPUT_TYPE_LOADERS[output_type](
                 IFACE, Mock(), self.session, self.hostname, calc_id,
                 output_type, filepath)
+            if dlg.ok_button.isEnabled():
+                dlg.accept()
+                print('\t\tok')
```

Comment:
Why is this print needed?

Reply:
Is it printed when running the tests? If yes, please add a comment saying so.

Reply:
From the context, I suppose it is already clear enough. There are many prints of that kind in the workflow, like "Loading output X... ok"